### DISCUSSION TITLE
针对API请求路径，可以考虑增加可配置的路径前缀

### DISCUSSION THREAD

**Discussion by dev4u** [OwO-Network/DLX#122] [Ideas]
如题，相比较固定的API请求路径，增加一个隐藏的路径前缀，可以避免被爬虫挖出大量不希望被公开的deepLx资源。

**Comment by missuo** [comment]
You can view the following document. DeepLX supports access token, which you can use to protect your API from abuse.

https://deeplx.owo.network/install/variables.html

**Reply by dev4u** [reply]
我觉得如果能设置一个前缀，也许可以更好的隐藏服务。
